recipes/jinja2cpp/all/conanfile.py

```diff
@@ -76,6 +76,12 @@ def validate(self):
             raise ConanInvalidConfiguration(
                 f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support."
             )
+        if Version(self.version) >= "1.3.1" and self.settings.compiler == "apple-clang":
```

Comment:
I have a special switch for this case
https://github.com/jinja2cpp/Jinja2Cpp/blob/master/CMakeLists.txt#L18

for apple-clang there can be -DJINJA2CPP_WITH_JSON_BINDINGS=rapid, which doesn't require boost::json

Reply:
@rmorozov 
Thanks a lot!
I will fix this issue.

Reply:
@rmorozov 
I tried to compile jinja2cpp/1.3.1 with apple-clang with INJA2CPP_WITH_JSON_BINDINGS=rapid.
But I get the same compile error due to Boost::json.
This is because CMakeLists.txt in jinja2cpp/1.3.1 tries to link Boost::json despite the value of INJA2CPP_WITH_JSON_BINDINGS.
I will try to make jinja2cpp/1.3.1 not compatible with apple-clang in this PR, because I think it is necessary to fix CMakeLists.txt or conan's boost recipe.